internal/pkg/externalplugins/owners/owners_test.go

```diff
@@ -55,6 +61,7 @@ func (f *fakegithub) ListTeamMembers(_ string, teamID int, role string) ([]githu
 	teams := map[int][]github.TeamMember{
 		0:  {{Login: "default-sig-lead"}},
```

Comment:
Could you please use different MEMBERS to make the test results more distinguishable?

Like:
0: admin1,admin2
42: sig-leader1,sig-leader2
60: releaser1,releaser2

Reply:
ok

Reply:
reset
/lgtm